platform/search/src/services/GenericSearchProvider.js

```diff
@@ -273,7 +277,9 @@ define([
 
             modelResults.hits = event.data.results.map(function (hit) {
                 return {
-                    id: hit.id
+                    id: hit.id,
+                    name: hit.name,
+                    type: hit.type
                 };
```

Comment:
Is there any reason we need to go over the hits and return these items? Seems the hits have the properties we need right? It would save us going over the array